released/packages/coq-hierarchy-builder/coq-hierarchy-builder.1.7.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+synopsis:
+  "High level commands to declare and evolve a hierarchy based on packed classes"
+description: """\
+Hierarchy Builder is a high level language to build hierarchies of algebraic structures and make these
+hierarchies evolve without breaking user code. The key concepts are the ones of factory, builder
+and abbreviation that let the hierarchy developer describe an actual interface for their library.
+Behind that interface the developer can provide appropriate code to ensure retro compatibility."""
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: ["Cyril Cohen" "Kazuhiko Sakaguchi" "Enrico Tassi"]
+license: "MIT"
+tags: "logpath:HB"
+homepage: "https://github.com/math-comp/hierarchy-builder"
+bug-reports: "https://github.com/math-comp/hierarchy-builder/issues"
+depends: [
+  "coq-elpi" {(>= "2.0") | = "dev"}
+]
+conflicts: ["coq-hierarchy-builder-shim"]
+build: [
+  [make "build"]
```

Comment:
Is there a reason to not use make-level parallelism here?

Reply:
I don't think there is

Reply:
OK, let's try and see what CI says.